examples/sagemaker-tensorflow/sagemaker-merlin-tensorflow.ipynb

```diff
@@ -0,0 +1,1392 @@
+{
```

Comment:
<p>I get the error <code>/bin/bash: aws: command not found</code>. awscliv2 was installed but there was no alias made calling it aws. using awsv2, however, results in error.</p><pre class="ql-syntax" spellcheck="false">18:31:02 - awscliv2 - ERROR - Command failed with code 127
</pre><p>Alternatively under the pip installs on top of the notebook you can add:</p><pre class="ql-syntax" spellcheck="false">!curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
!unzip awscliv2.zip
!./aws/install
</pre><p>Or boto3 can be used to download the data. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/692/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Ah, good idea. Removing awscli and using boto3 might be cleaner.